sickbeard/providers/scenetime.py

```diff
@@ -87,7 +85,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     logger.log('Search string: {search}'.format
                                (search=search_string), logger.DEBUG)
 
-                search_url = self.urls['search'] % (quote(search_string), self.categories)
+                search_url = self.urls['search'].format(search_string, self.categories)
```

Comment:
I think this still needs to be quoted because its being passed as part of the URL.  Really the cats and search string should be converted to params to pass instead and then the quote would be unnecessary.